po/README.md

```diff
@@ -0,0 +1,19 @@
+# Translations generated by Launchpad
+
+## How to sync translations from Launchpad?
+- Run `update-pot` to generate latest translation template (`po/snappy.pot`)
+- Import generated translation template to snappy entry [here](https://translations.launchpad.net/snapd/trunk/+templates)
+- Wait until import is approved. You can monitor status in the import queue [here](https://translations.launchpad.net/snapd/trunk/+imports)
+- Export latest translations [here](https://translations.launchpad.net/snapd/trunk/+export) (download link will be sent to your email)
```

Comment:
Consider using this cleaner way of referencing urls: https://github.com/snapcore/snapd/blob/master/CONTRIBUTING.md?plain=1#L161-L162

Reply:
updated